### PR TITLE
rewrite image_uploader.js in vanilla js

### PR DESCRIPTION
**Pull request by goapunk**

### CODE REVIEW

File: adhocracy4/images/assets/image_uploader.js [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+          text.value = name
+        }
         previewImage.alt = name
         if (window.FileReader) {
           const reader = new window.FileReader()
-          reader.onload = function (e) {
-            previewImage.attr('src', e.target.result)
+          reader.addEventListener('load', function (e) {
+            previewImage.setAttribute('src', e.target.result)
             clearInput.prop('checked', false)
```
Comment (khamui) [inline adhocracy4/images/assets/image_uploader.js:22]: in line 26 you rewrote a similar assignment, maybe you want to do it here as well?

**Review by khamui — COMMENTED** [review]
nice! just one comment

File: adhocracy4/images/assets/image_uploader.js (hunk 1 above)
Comment (goapunk) [reply adhocracy4/images/assets/image_uploader.js:22]: oh, good catch `prop` is still jquery :facepalm:

**Review by khamui — APPROVED** [review]
cool! looking good from reading code.

I can't test it because currently i am having a4 linking issues

**Review by fuzzylogic2000 — APPROVED** [review]
Yeah, I tried it out, it works!
But I needed a while to find out how we use it (through the images.widgets.ImageInputWidget, which is used in images.forms.ImageField, which is then used in images.fields.ConfiguredImageField, which we use for all images in the model files of mB and probably also elsewhere). I don't know if that should go into a comment or docstring somewhere?

**Comment by goapunk** [comment]
> Yeah, I tried it out, it works! But I needed a while to find out how we use it (through the images.widgets.ImageInputWidget, which is used in images.forms.ImageField, which is then used in images.fields.ConfiguredImageField, which we use for all images in the model files of mB and probably also elsewhere). I don't know if that should go into a comment or docstring somewhere?

sounds like a good idea, do you want to add it? You actually investigated the usage much better than I did :sweat_smile:

**Comment by fuzzylogic2000** [comment]
> > Yeah, I tried it out, it works! But I needed a while to find out how we use it (through the images.widgets.ImageInputWidget, which is used in images.forms.ImageField, which is then used in images.fields.ConfiguredImageField, which we use for all images in the model files of mB and probably also elsewhere). I don't know if that should go into a comment or docstring somewhere?
> 
> sounds like a good idea, do you want to add it? You actually investigated the usage much better than I did sweat_smile

Ah, another time. :p

**Pull request merged**